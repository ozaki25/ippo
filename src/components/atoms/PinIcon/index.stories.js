import React from 'react';
import PinIcon from '.';

export default {
  title: 'atoms/PinIcon',
};

const props = {};

export const 通常パターン = () => <PinIcon {...props} />;
