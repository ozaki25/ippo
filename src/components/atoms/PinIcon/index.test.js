import React from 'react';
import { snapshot } from 'src/test/helpers';
import PinIcon from '.';

const props = () => ({});

snapshot('PinIcon/nomal', <PinIcon {...props} />);
