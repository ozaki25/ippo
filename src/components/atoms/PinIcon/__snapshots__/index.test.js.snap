// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: PinIcon/nomal 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <MockRouter
    createHref={[Function]}
    location=""
    params={Object {}}
    path=""
    push={[Function]}
    replace={[Function]}
    url=""
  >
    <PinIcon>
      <WithStyles(SvgIcon)>
        <SvgIcon
          classes={
            Object {
              "colorAction": "MuiSvgIcon-colorAction-4",
              "colorDisabled": "MuiSvgIcon-colorDisabled-6",
              "colorError": "MuiSvgIcon-colorError-5",
              "colorPrimary": "MuiSvgIcon-colorPrimary-2",
              "colorSecondary": "MuiSvgIcon-colorSecondary-3",
              "fontSizeInherit": "MuiSvgIcon-fontSizeInherit-7",
              "fontSizeLarge": "MuiSvgIcon-fontSizeLarge-9",
              "fontSizeSmall": "MuiSvgIcon-fontSizeSmall-8",
              "root": "MuiSvgIcon-root-1",
            }
          }
          color="inherit"
          component="svg"
          fontSize="default"
          viewBox="0 0 24 24"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-1"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19.083 1c0 1.018-1.424 1.907-3.541 2.382V11c2.926.652 4.958 2.086 4.958 3.751h-7.792V23h-1.416v-8.25H3.5c0-1.665 2.032-3.1 4.958-3.751V3.382C6.341 2.907 4.917 2.018 4.917 1h14.166z"
            />
             
          </svg>
        </SvgIcon>
      </WithStyles(SvgIcon)>
    </PinIcon>
  </MockRouter>
</Provider>
`;
